unity-client/Assets/Scripts/MainScripts/DCL/Controllers/WebRequest/WebRequestAssetBundle.cs

```diff
@@ -1,12 +1,40 @@
+using UnityEngine;
 using UnityEngine.Networking;
 
 namespace DCL
 {
+    /// <summary>
+    /// Our custom implementation of the UnityWebRequestMultimedia (Audio Clip).
```

Comment:
This is for Audio Clip? I think this is a typo, it should be asset bundle right?

Reply:
You're right! Thanks! Fixed in https://github.com/decentraland/explorer/pull/2256/commits/142347c71fa9f48bb2c13e48e06a209dca2f86d1